main.nf

```diff
@@ -99,13 +129,32 @@ workflow {
 
 }
 
+
+process ParseCsv {
+  container "$container__pandas"
```

Comment:
It's not important here, but I like to put these in the `params` scope in `nextflow.config` to make them easily overridable on the command line for future testing.

Reply:
I like this pattern. I'll make a note to include it in the next time I touch this code.